Sources/Bow/Data/Try.swift

```diff
@@ -3,9 +3,11 @@ import Foundation
 /// Models an error in an invocation to some methods of `Try`.
 ///
 /// - illegalState: The value is in an illegal state when an operation is invoked.
+/// - predicateDoesNotMatch: The value does not match the provided predicate.
```

Comment:
```suggestion
/// - predicateError: The value does not match the provided predicate.
```

Reply:
I think the current name is more specific to the error and the suggestion is too generic.

Reply:
I agree, but then I think we should change the `case` option, wdut?